### PR TITLE
feat: add group in the log when is multi-group

### PR DESCRIPTION
**Pull request by camilamacedo86**
Closes: #1383

### CODE REVIEW

File: testdata/project-v2-addon/go.mod [hunk 1]
```diff
@@ -6,8 +6,10 @@ require (
...
-	k8s.io/apimachinery v0.17.2
-	k8s.io/client-go v0.17.2
-	sigs.k8s.io/controller-runtime v0.5.0
-	sigs.k8s.io/kubebuilder-declarative-pattern v0.0.0-20200415210853-85eb326a6add
+	gonum.org/v1/netlib v0.0.0-20190331212654-76723241ea4e // indirect
+	k8s.io/apimachinery v0.18.2
+	k8s.io/client-go v0.18.2
+	sigs.k8s.io/controller-runtime v0.6.0
+	sigs.k8s.io/kubebuilder-declarative-pattern v0.0.0-20200507203943-d901af431e3a
+	sigs.k8s.io/structured-merge-diff v1.0.1-0.20191108220359-b1b620dd3f06 // indirect
```
Comment (Adirio) [inline testdata/project-v2-addon/go.mod:14]: Does these changes belong to this PR? Updating some library versions and introducing some indirect imports seem not to be part of it.

File: testdata/project-v3-addon/go.mod [hunk 2]
```diff
@@ -6,8 +6,10 @@ require (
...
-	k8s.io/apimachinery v0.17.2
-	k8s.io/client-go v0.17.2
-	sigs.k8s.io/controller-runtime v0.5.0
-	sigs.k8s.io/kubebuilder-declarative-pattern v0.0.0-20200415210853-85eb326a6add
+	gonum.org/v1/netlib v0.0.0-20190331212654-76723241ea4e // indirect
+	k8s.io/apimachinery v0.18.2
+	k8s.io/client-go v0.18.2
+	sigs.k8s.io/controller-runtime v0.6.0
+	sigs.k8s.io/kubebuilder-declarative-pattern v0.0.0-20200507203943-d901af431e3a
+	sigs.k8s.io/structured-merge-diff v1.0.1-0.20191108220359-b1b620dd3f06 // indirect
```
Comment (Adirio) [inline testdata/project-v3-addon/go.mod:14]: Same

File: testdata/project-v2-addon/go.mod (hunk 1 above)
Comment (camilamacedo86) [reply testdata/project-v2-addon/go.mod:14]: I am looking for the lgtm in the CI/master fix : https://github.com/kubernetes-sigs/kubebuilder/pull/1504

File: testdata/project-v3-addon/go.mod (hunk 2 above)
Comment (camilamacedo86) [reply testdata/project-v3-addon/go.mod]: Solved

File: testdata/project-v2-addon/go.mod (hunk 1 above)
Comment (camilamacedo86) [reply testdata/project-v2-addon/go.mod]: solved

**Comment by Adirio** [comment]
Wait, this can only be done for v3 onwards, v2 projects can be changed that way or they will be backwards incompatible.

**Comment by camilamacedo86** [comment]
HI @Adirio, 

The change here is only to add the Group in the log. It is not changing the structure/layout of the projects. So, it is not a breaking change at all.

**Comment by Adirio** [comment]
As we spoke in Slack, this is not technically bacwards incompatible, but the behaviour is a bit different. Previously scaffolded controllers won't have the group info while new ones will. I'm ok with this but let's see what @DirectXMan12 and @mengqiy have to say about it. Otherwise, lgtm

**Comment by camilamacedo86** [comment]
It will be addressed just for V3 in : https://github.com/kubernetes-sigs/kubebuilder/pull/1498

**Pull request closed without merging**